ui/nuxeo-routing-behavior.js

```diff
@@ -71,22 +71,26 @@ export const RoutingBehavior = {
   _computeUrlFor() {
     return function(...args) {
       if (this.router) {
-        const route = args[0];
-        const baseUrl = this.router.baseUrl || '';
-        if (route.startsWith('/')) {
-          return baseUrl + route;
+        const [route, ...params] = args;
+        if (!route) {
+          return '';
```

Comment:
we want to return an empty url here ?

Reply:
What would we return instead? We can return undefined, but this function is expected to return a string anyway.

This was added for those cases where you have a finding `[[urfFor(doc)]]` but doc is still `undefined`.

Reply:
We can try to fix this on `nuxeo-home`, which is the only place by default where we generate URLs from docs stored on local storage, AFAIK. But I'm not sure I'd call this robust.

Reply:
makes sense, just wondering why we did not have it before